jobs/cg.yml

```diff
@@ -0,0 +1,70 @@
+name: $(date:yyyyMMdd)$(rev:.r)
+trigger:
+  branches:
+    include:
+    - main
+
+schedules:
+- cron: 30 5 * * 0
+  branches:
+    include:
+    - main
+  always: true
+
+resources:
+  repositories:
+  - repository: MicroBuildTemplate
+    type: git
+    name: 1ESPipelineTemplates/MicroBuildTemplate
+    ref: refs/tags/release
+
+extends:
+  template: azure-pipelines/MicroBuild.1ES.Official.yml@MicroBuildTemplate
+  parameters:
+    pool:
+      name: AzurePipelines-EO
+      image: AzurePipelinesWindows2022compliantGPT
+      os: windows
+    sdl:
+      sourceAnalysisPool:
+        name: AzurePipelines-EO
+        image: AzurePipelinesWindows2022compliantGPT
+        os: windows
+      policheck:
+        enabled: true
+      tsa:
+        enabled: false
+    featureFlags:
+      autoBaseline: false
+
+    stages:
+    - stage: build
+      jobs:
+      - job: Phase_1
+        displayName: Build cpptools.vsix
```

Comment:
Is this correct? Isn't cpptools.vsix only built by vscode-cpptools?

Reply:
Thanks.  I copy/pasted from vscode-cpptools and missed this one.